models/demos/t3000/mixtral8x7b/tt/mixtral_common.py

```diff
@@ -168,6 +168,24 @@ def preprocess_inputs_prefill(input_prompts, tokenizer, model_args, dtype, instr
     )
 
 
+def get_chunk_size(s):
```

Comment:
Add a comment to this function to explain where/what for we use these chunk sizes

Reply:
Removed this function because it is no longer needed (it was used in flash attention v1)